code/modules/mob/living/simple_mob/subtypes/animal/passive/possum.dm

```diff
@@ -196,7 +196,7 @@
 	icon_living = "poppy"
 	icon_dead = "poppy_dead"
 	icon_rest = "poppy_dead"
-	tt_desc = "Didelphis astrum salutem"
```

Comment:
I think this an entirely deliberate joke where shes specifically a Safety possum. I don't think anybody's going to get thrown off by a subtle latin joke being in there.

Reply:
yeah weighed whether or not to be the fun police. i will stop being the fun police though we definitely want `E` in there